README.md

```diff
@@ -143,4 +143,29 @@ Afterwards, you can deploy a target cluster.
 If you are running tilt on a remote machine, you can forward the web interface
 by adding the following parameter to the ssh command `-L 10350:127.0.0.1:10350`
 
-Then you can access the Tilt dashboard locally [here](http://127.0.0.1:10350)
\ No newline at end of file
+Then you can access the Tilt dashboard locally [here](http://127.0.0.1:10350)
+
+### Recreating local ironic containers
+
+In case, you want recreate the local ironic containers enabled with TLS, you
```

Comment:
```suggestion
In case, you want recreate the local ironic containers enabled with TLS and Basic Auth, you
```

Reply:
Thanks. Will take the change in another PR which I am doing. Don't want to waster two more CI runs on this.